apps/main-chain/src/eventListeners/colony.ts

```diff
@@ -175,6 +171,12 @@ export const setupListenersForColony = (
     ),
   );
 
+  addProxyColoniesEventListener(
+    ContractEventsSignatures.ProxyColonyRequested,
+    colonyAddress,
+    handleProxyColonyRequested,
+  );
```

Comment:
This introduces a bug for all create proxy colony actions to be added to `planex`, but will be solved in the future by some network changes